### PR TITLE
Tech 556

### PR DESCRIPTION
**Pull request by cjmabry**

### CODE REVIEW

File: pages/static/src/scss/helpers/_css-helpers.scss [hunk 1]
```diff
@@ -1138,6 +1138,22 @@ a.white-text {
   }
 }
 
+.web-store-bg {
```
Comment (ericbroder) [inline pages/static/src/scss/helpers/_css-helpers.scss]: Suggest we try and have more standard naming conventions in future. SMACSS is pretty good https://smacss.com/

Comment (cjmabry) [reply pages/static/src/scss/helpers/_css-helpers.scss]: We unofficially use BEM http://getbem.com/ but should solidify and document in near future.

Comment (ericbroder) [reply pages/static/src/scss/helpers/_css-helpers.scss]: Ok interesting, I'm not familiar with BEM. Yeah let's revisit at some point and get on same page.

Comment (ericbroder) [reply pages/static/src/scss/helpers/_css-helpers.scss]: @cjmabry made ticket https://ourrevolution.atlassian.net/browse/TECH-558

**Pull request merged**